modules/cmp/initialize.go

```diff
@@ -116,7 +116,7 @@ func do() (*httpserver.Server, error) {
 	bundleOpts := []bundle.Option{
 		bundle.WithHTTPClient(
 			httpclient.New(
-				httpclient.WithTimeout(time.Second, time.Second*60),
+				httpclient.WithTimeout(30*time.Second, time.Second*60),
```

Comment:
To unify the format, either all are `time.Second * num`, or all are `num * time.Second`
```go
httpclient.WithTiemeout(time.Second * 30, time.Second * 60)
```